packages/stk/stk_io/stk_io/StkMeshIoBroker.cpp

```diff
@@ -100,2888 +102,1255 @@
 // clang-format on
 // #######################   End Clang Header Tool Managed Headers  ########################
 
-
-namespace {
-  const std::string entity_nodes_suffix = "_n";
-} //namespace <empty>
-
 namespace stk {
 namespace io {
 
-  template <typename T>
-  bool is_index_valid(const std::vector<T> &file_vector, size_t input_file_index)
-  {
+template <typename T>
+bool is_index_valid(const std::vector<T> &file_vector, size_t input_file_index)
+{
     bool invalid = file_vector.empty() ||
-      input_file_index >= file_vector.size() ||
-      Teuchos::is_null(file_vector[input_file_index]);
+            input_file_index >= file_vector.size() ||
+            Teuchos::is_null(file_vector[input_file_index]);
```

Comment:
It's not really just any old `T`; it's actually something more like `Teuchos::RCP<T>`, right?  It would be better to constrain `T` in the interface than in the implementation.